source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -4,43 +4,75 @@ title: Using Node.js at GDS
+The advice here is specifically about Node.js. Generic guidelines on writing
+code are out of scope.
 
-```
-❯ Running tasks for *.js
-   ✖ standard --fix
-     → standard: Use JavaScript Standard Style (https://standardjs.com)
-     git add
-✖ standard --fix found some errors. Please fix them and try committing again.
-  ~/nodejs-starter/index.js:5:9: 'hello' is assigned a value but never used.
+If you want to contribute to this document please see the [Updating this
+manual](#updating-this-manual) section below.
 
-```
+Most guidelines listed here are recommendations, and the authors acknowledge
+that there will sometimes be valid exceptions.
+
+## Node versions
+
+> Only use Long Term Support (<abbr title="Long term support">LTS</abbr>)
+versions of Node.js.
+
+These are even-numbered versions (for example, Node.js 6.x or 8.x). However, it is important
+to keep an eye on the [Node.js LTS Schedule] as to when versions move in and out
+of LTS.
+
+> Don't use the `--harmony` or in-progress feature flags
+
+*Staged* or *in-progress* features aren't considered stable by the Node implementors.
+
+> Don't use language features beyond EcmaScript2016
 
-#### Editor plugins
-It’s useful to highlight lint errors before starting the commit process. To do this you can get live linting in your text editor of choice by installing one of [Standard’s plugins](https://standardjs.com/#are-there-text-editor-plugins). Users of [SublimeLinter](http://www.sublimelinter.com/) will find [StandardJS] will work out of the box using the default config.
+Even though ES2017 is final, some features aren't yet implemented in LTS versions. Check using [node.green](http://node.green/).
 
-### Structure
-As well as having a consistent code style, it is important that our projects and repositories are structured in similar ways. This allows developers to find things quickly and easily. We want our code to be modular and reusable so our project structure should reflect this. Below are our four guiding principles: (thanks to [Rising Stack](https://blog.risingstack.com/) for this [blog post](https://blog.risingstack.com/node-hero-node-js-project-structure-tutorial/)).
+## Source formatting and linting
+
+> Lint your JavaScript code with [XO]
+
+We use the JavaScript style used by [XO] to make all our codebase use the same standard. However we make two exceptions: we disallow semicolons and we mandate spaces over tabs. Your `package.json` file should [reflect these exceptions](https://github.com/alphagov/nodejs-starter/blob/master/package.json#L24).
```

Comment:
I would like to propose that we allow semicolons as the TC39 is now [recommending there usage](https://github.com/tc39/ecma262/pull/1062). It is huge PR with a tonne of comments so it is a controversial topic. As we look to be moving away from Standard this seems like the ideal time to make this change too.

I wouldn't expect this to be a retroactive change, but it should be used for new projects going forwards.

Reply:
Worth to mention: `xo --fix` to be your friend.